web/beacon-app/src/components/ui/Tag/Tag.tsx

```diff
@@ -2,7 +2,7 @@ import { mergeClassnames } from '@rotational/beacon-core';
 import React from 'react';
 export interface TagProps {
   children: React.ReactNode;
-  variant?: 'primary' | 'success' | 'warning' | 'error' | 'ghost' | 'secondary';
+  variant?: 'primary' | 'success' | 'warning' | 'error' | 'ghost' | 'secondary' | string;
```

Comment:
can you remove `string` at the end of the variant?

Reply:
When I added the `topicStateMap` in the `TopicStateTag`, variant returned an error because none of the selected options are listed. This is why I added string here. Is there some other way to get variant to work with the `topicStateMap` that I've somehow missed?

Reply:
Never mind. Looks like all I had to do was add const after the map.